src/css/SocialsIcon.module.css

```diff
@@ -3,17 +3,24 @@
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
-  width: 129px;
+  width: max-content;
+}
+
+.iconLink {
+  margin-right: 0.75rem;
 }
 
 .iconLink:hover {
```

Comment:
For the sake of consistency and accessibility, can we also apply this to `.iconLink:focus`?

If we change it to:

```css
.iconLink:hover,
.iconLink:active {
```

It will also apply the opacity when keyboard tab navigation is used :)

Reply:
Ah good call, I didn't realize there wasn't a `focus` state for this element yet. Will change this later! Thanks for catching it 🚀